src/connectors/search-box/connectSearchBox.js

```diff
@@ -82,7 +81,7 @@ export default function connectSearchBox(renderFn, unmountFn) {
         this._clear();
       },
 
-      init({ helper, onHistoryChange, instantSearchInstance }) {
+      init({ helper, instantSearchInstance }) {
```

Comment:
technically a breaking change; but practically this should be fine

Reply:
Yes, the fact that we decided not to document this motivated me to remove it. Even the current team wasn't aware of `onHistoryChange`.